docs/src/componentDocs/Header/playground/PlaygroundPage.tsx

```diff
@@ -0,0 +1,271 @@
+        required: false,
+        initialValue: 'static',
+        options: ['static', 'dynamic'],
+        category: 'Optional Props',
+        disabled: true,
+    },
+    {
+        id: 'fontColor',
+        type: 'color',
+        typeLabel: 'string',
+        description: 'Color of the title, subtitle, info, and icons in the header',
+        required: false,
+        initialValue: '',
+        category: 'Optional Props',
+    },
+    {
+        id: 'info',
+        type: 'string',
+        typeLabel: 'string',
+        description: 'Third line of text (hidden on collapse)',
+        required: false,
+        initialValue: 'Text hidden on collapse',
+        category: 'Optional Props',
+    },
+    {
+        id: 'icon',
+        type: 'select',
+        typeLabel: 'IconSource',
+        description: 'Icon to show to the left of the title',
+        initialValue: '',
+        options: [
+            { label: 'undefined', value: 'undefined' },
+            { label: 'menu', value: 'menu' },
+        ],
+        required: false,
+        category: 'Optional Props',
+    },
+    {
+        id: 'navigationIconColor',
+        type: 'color',
+        typeLabel: 'string',
+        description: 'The color used for the Navigation icon',
+        required: false,
+        initialValue: '',
+        category: 'Optional Props',
+    },
+
+    // Other Configuration
+    // NONE
+];
+
+const HeaderPreview: PreviewComponent = ({ data }) => {
+    const { icon, backgroundImage, actionItems, ...rest } = data as unknown as HeaderProps;
+    const containerRef = useRef(null);
+    const SCROLL_CONTAINER_ID = 'playground-scroll-container';
+    const getIcon = (value: string): IconSource | undefined => {
+        switch (value) {
+            case 'menu':
+                return { name: 'menu' };
+            case 'undefined':
+            default:
+                return undefined;
+        }
+    };
+    const getActionItems = (value: boolean): any => {
+        switch (value) {
+            case true:
+                return [{ icon: { name: 'settings' } }];
+            default:
+                return undefined;
+        }
+    };
+
+    return (
+        <Box
+            sx={{
+                m: '16px 0',
+                maxHeight: '100%',
+                maxWidth: '100%',
+                position: 'relative',
+            }}
+            ref={containerRef}
+        >
+            <Header
+                style={{ width: DRAWER_WIDTH, margin: 'auto' }}
+                {...rest}
+                title={'Valley Forge'}
+                subtitle={'The Last Stand'}
+                icon={getIcon(icon as unknown as string)}
+                actionItems={getActionItems(actionItems as unknown as boolean)}
+                // searchableConfig={{ onChangeText: () => {} }}
+                backgroundImage={getImage(backgroundImage?.toString() ?? '')}
+                variant="static"
+            />
```

Comment:
add the following styles `styles={{backgroundImage:{width: DRAWER_WIDTH, margin: 'auto' }}}`